src/applications/mhv-medications/containers/Prescriptions.jsx

```diff
@@ -144,18 +144,15 @@ const Prescriptions = () => {
     scrollLocation?.current?.scrollIntoView();
   };
 
-  useEffect(
-    () => {
-      if (!isLoading) {
-        if (prescriptionId) {
-          goToPrevious();
-        } else {
-          focusElement(document.querySelector('h1'));
-        }
+  useEffect(() => {
+    if (!isLoading) {
+      if (prescriptionId) {
+        goToPrevious();
+      } else {
+        focusElement(document.querySelector('h1'));
       }
-    },
-    [isLoading, prescriptionId],
-  );
+    }
+  }, []);
```

Comment:
Curious as to why you took out the dependancies here?

Reply:
So that useEffect runs only once on page load. Scroll to previous medication only happens once on load when user goes back from Medication Details, focusing `h1` only happens on page load (unlike it was triggered on Sort before).

Reply:
I see, appreciate the clarification 👍